src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Browser SPA monitoring 2.0
+tags:
+  - Browser
+  - Single page app monitoring
+translate:
+  - jp
+metaDescription: The new browser SPA monitoring capability is a streamlined way to monitor your users' experience and app performance.
+freshnessValidatedDate: 02-01-2024
+---
+
+We're excited to announce an overhaul of our Pro+SPA agent, built to address numerous pain points voiced by our customers. This update tackles critical limitations of the current agent, paving the way for a smoother, more optimized, and future-proof SPA monitoring experience.
```

Comment:
Not sure if this distinction should be made, but we're not changing the Pro+SPA agent per se, rather the SPA feature inside it and only that.